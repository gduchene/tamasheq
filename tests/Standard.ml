let ( $ ) f x = f x

let () = print_endline "Hello world!"
let () = print_int $ List.length [1; 2; 3] |> print_newline
let () = Printf.printf "The length of [1; 2; 3] is %d\n" $ List.length [1; 2; 3]
