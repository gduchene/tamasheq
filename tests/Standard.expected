Hello world!
3
The length of [1; 2; 3] is 3
Standard_0	=	[1] {
    (no name)	=	<abstr:247>
}
